code/fundamentals.rb

```diff
@@ -8,8 +8,16 @@
 
 # 1. Create a variable `my_favorite_animal` and assign it the name of your favorite animal
 
+puts my_favorite_animal = "Tiger"
+
 # 2. Print "My favorite animal is <your-favorite-animal>" by printing a string with `my_favorite_animal` interpolated into it.
 
+print "My favorite animal is" + "my_favorite_animal"
```

Comment:
close, no quotes around my favorite animal so `puts "My favorite animal is" + my_favorite_animal`. Also, prompt asks for interpolation so `puts "My favorite animal is #{my_favorite_animal}"`